### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by niallobrien** [nooljs/nooljs#1]
Hi, has this project been abandoned?

Thanks in advance.

**Comment by cgkrish** [comment]
No,  Since I am currently busy with my office tasks   I did not contribute much  to this project recently.
 I am planning to migrate it to support Angular 2.0 and some of improvement.
If you want to give feedback  or any issues then I can add into this.
Thanks
